addons/mail/static/src/scss/emojis.scss

```diff
@@ -13,11 +13,19 @@
     }
 }
 
+.o_form_view {
+    // Widget(emojis) should be hidden when field has "invisible modifier" !
+    .o_invisible_modifier + .o_mail_add_emoji{
+        display: none !important;
+    }
+}
+
 .o_mail_emojis_dropdown {
```

Comment:
Can we use the `position-relative` bootstrap class on the element instead? Or is that not an option?
We try to use as many bootstrap classes as possible to avoid custom css (easier to see what's applied when you read the HTML).